### DISCUSSION TITLE
Difficulty Locating Core Dump Files and Permission Issues with core_pattern.

### DISCUSSION THREAD

**Discussion by tom-mad** [mamba-org/micromamba-docker#391] [Q&A] [answered]
I am unable to locate core dump files even when expecting them. Additionally, I am experiencing difficulties changing the /proc/sys/kernel/core_pattern due to permission constraints. Even with root privileges, I do not have the necessary access. The generated core dump files are expected after executing commands such as:
```
micromamba install -c conda-forge cxx-compiler
ulimit -c unlimited
echo "int main(){int * a; *a=10;}" > main.cpp
g++ main.cpp
./a.out
find / -name "*core.*" -type f -mmin -5
```

**Comment by maresb** [comment]
Hi! Have you tried `docker run --user=root ...`?

**Reply by tom-mad** [reply]
Hi, thank you for answer, in any situation I tested when i want save something to core_pattern i get message "bash: core_pattern: Read-only file system"

**Reply by maresb** [reply]
Does this work in any other Docker containers? This seems to me more likely a Docker issue than a micromamba-docker issue.

I don't know much about the technical details, but Docker shares the same Linux kernel as the host. If you're trying to adjust the kernel settings, then you should probably do that on the host system instead of in the container. Similarly, if the kernel is generating dump files, then you might want to check for the files on your host system.

I know very little about this stuff, so apologies if I'm writing nonsense.

**Comment by wholtz** [comment, marked as answer]
Hello @tom-mad 

[This blog post](https://ddanilov.me/how-to-configure-core-dump-in-docker-container) indicates that `/proc/sys/kernel/core_pattern` should be configured on the host, not within the container.

**Reply by tom-mad** [reply]
Thank you very much for your help.
